src/drivers/CgreenDriver.cpp

```diff
@@ -0,0 +1,159 @@
+#include <cucumber-cpp/internal/drivers/CgreenDriver.hpp>
+
+#include <boost/bind.hpp>
+#include <boost/function.hpp>
+
+using namespace cgreen;
+
+namespace cucumber {
+namespace internal {
+
+
+bool CgreenStep::initialized = false;
+
+static const char* const blacklist[] = {
+        "Running",
+        "Completed",
+        "Failure",
+        "currentTest",
+        "\n",
+        "/"
+};
+
+class CukeCgreenInterceptor {
```

Comment:
The project doesn't seem to enforce c++11 so I can't assume std::unique_ptr is available.